CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 ### Changed
 
```

Comment:
I think we should document the `--https-port` in the added section as well but we should specifically call out it only works with the nginx plugin for now.

Reply:
Done.